File: programming files/datetime_convert.java
import java.time.LocalDateTime;
import java.time.format.DateTimeFormatter;

public class datetime_convert {
    public static void main(String[] args) {
        String dateStr = "2022-03-1710:45:30";
        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-mm-dd hh:mm:ss");
        LocalDateTime dateObj = LocalDateTime.parse(formatter, dateStr);
        String formattedDate = dateObj.format(DateTimeFormatter.ofPattern("MM/dd/yyyy HH:mm:ss"));

        System.out.println(formattedDate);
    }
}

File: programming files/datetime_convert.py
from datetime import datetime

date_str = "2022-03-17 10:45:30"
date_obj = datetime.strftime(date_str, '%Y-%m-%d %H:%m:%S')
formatted_date = date_obj.strftime('%m/%d/%Y %H:%m:%S')

print(formatted_date)
